client/src/test/App.test.js

```diff
@@ -1,21 +1,21 @@
-import { render } from "@testing-library/react";
 import { expect } from "chai";
+import { shallow, mount } from "enzyme";
 import React from "react";
-import ReactDOM from "react-dom";
 
 import App from "../App";
+te;
 
 /* eslint-disable */
 describe("Good display", () => {
   // Jest test
-  it("renders without crashing", () => {
-    const div = document.createElement("div");
-    ReactDOM.render(<App />, div);
+  it("renders without crashing", async () => {
+    const wrapper = shallow(<App />);
+    await wrapper.instance().componentDidMount();
   });
 
   // Chai test
   it("contains the good link", () => {
-    const { getByText } = render(<App />);
-    expect(getByText(/À propos/i)).to.be.not.null;
+    const wrapper = mount(<App />);
+    expect(wrapper.text()).to.match(/À Propos/i);
```

Comment:
I've solved the problem in another way in #206, but I really prefer your solution!